spring-session-data-redis/src/main/java/org/springframework/session/data/redis/ReactiveRedisSessionRepository.java

```diff
@@ -274,8 +274,13 @@ private boolean hasChangedSessionId() {
 			String sessionKey = getSessionKey(getId());
 			Mono<Boolean> update = ReactiveRedisSessionRepository.this.sessionRedisOperations.opsForHash()
 					.putAll(sessionKey, new HashMap<>(this.delta));
-			Mono<Boolean> setTtl = ReactiveRedisSessionRepository.this.sessionRedisOperations.expire(sessionKey,
-					getMaxInactiveInterval());
+			Mono<Boolean> setTtl;
+			if (getMaxInactiveInterval().getSeconds() >= 0) {
+				setTtl = org.springframework.session.data.redis.ReactiveRedisSessionRepository.this.sessionRedisOperations.expire(sessionKey,
```

Comment:
There is no need to add the full path for `ReactiveRedisSessionRepository`.

Reply:
I have completed the modification as required, please review